packages/grafana-ui/src/components/ColorPicker/NamedColorsGroup.tsx

```diff
@@ -26,7 +27,7 @@ const NamedColorsGroup: FunctionComponent<NamedColorsGroupProps> = ({
     <div className={styles.colorRow}>
       <div className={styles.colorLabel}>{label}</div>
       <div {...otherProps} className={styles.swatchRow}>
-        {hue.shades.map((shade) => (
+        {reverseMap(hue.shades, (shade) => (
```

Comment:
Using a utility function here instead of `Array.prototype.reverse` as I'd prefer to just iterate over the array in reverse, rather than mapping and _then_ reversing